README.md

```diff
@@ -1,2 +1,79 @@
 # template-library
 Template for NPM library.
+
+# Selectively Grammar
+## Symbols
+- **_bool.expr._** (boolean expression)
+- **_id._** (identifier or combination of identifiers, e.g. identifier.identifier)
+- **_expr._** (expression)
+- **_expr.op._** (expression operator e.g. >, <, >=, <=)
+- **_alg.expr._** (algebraic expression)
+- **_bin.op._** (binary operator)
+
+## Production Rules
+
+### Boolean Rules:
+- **_bool.expr._** ::== **_id._** ":" **_expr._** | **_id._** **_expr.op._** **_expr._** | (**_bool.expr._**) | **_bool.expr._** "|" **_bool.expr._** | **_bool.expr._** **_bool.expr._** | !**_bool.expr._**  
+- **_id._** ::== **_id._** "." identifier | identifier  
+- **_expr._** ::== **_id._** | string | **_alg.expr._** | (**_expr._**) | "!"**_expr._** | **_expr._** "|" **_expr._**    
+- **_expr.op._** ::== < | > | <= | >=  
+### Algebraic Rules:  
+- **_alg.expr._** ::== number | **_alg.expr._** **_bin.op._** **_alg.expr._** | (**_alg.expr._**)  
+- **_bin.op._** ::== + | - | / | *  
```

Comment:
Looks kind of wordy. "." at the end of Symbols seems unnecessary. Maybe better terms for it. Either by defining short terms for them or use long terms. Might be clearer even with longer rules.